pkg/autoscaler/collector.go

```diff
@@ -271,25 +291,40 @@ func (c *collection) currentMetric() *av1alpha1.Metric {
 // record adds a stat to the current collection.
 func (c *collection) record(stat Stat) {
 	// Proxied requests have been counted at the activator. Subtract
-	// AverageProxiedConcurrentRequests to avoid double counting.
-	c.buckets.Record(*stat.Time, stat.PodName, stat.AverageConcurrentRequests-stat.AverageProxiedConcurrentRequests)
+	// them to avoid double counting.
+	c.concurrencyBuckets.Record(*stat.Time, stat.PodName, stat.AverageConcurrentRequests-stat.AverageProxiedConcurrentRequests)
+	c.opsBuckets.Record(*stat.Time, stat.PodName, stat.RequestCount-stat.ProxiedRequestCount)
```

Comment:
Do we want to divide by the time from the last recorded bucket, or we'll hope that 1s scrapes are precise enough?

Reply:
`RequestCount` is counted and reset every 1 second by queue-proxy or activator. So we can use them as OPS. There's work divide the data by the reporting period time so that if we change report period from 1 second to whatever, we can still use them as OPS: https://github.com/knative/serving/issues/2524

In collector side, I don't think it makes sense to divide them by the time from the last recorded bucket as they are "request counts over the last 1 second" instead of "request counts over the time from the last recorded".

Reply:
I agree with @yanweiguo here. That'll be done on the reporting side.